# remove IFLA_* enums missing in 'if_link.h' of some toolchains.
# 
--- src/enum_net.cpp.orig	2020-08-20 21:04:27.000000000 +0000
+++ src/enum_net.cpp	2022-03-26 22:20:17.501255676 +0000
@@ -337,8 +337,7 @@
 				case IFLA_LINK: memcpy(&ret.type, ptr, sizeof(int)); break;
 				case IFLA_OPERSTATE: memcpy(&ret.oper_state, ptr, sizeof(int)); break;
 
-				// ignore these attributes
-				case IFLA_CARRIER:
+				// ignore these (and other) attributes
 				case IFLA_ADDRESS:
 				case IFLA_BROADCAST:
 				case IFLA_QDISC:
@@ -351,7 +350,6 @@
 				case IFLA_LINKINFO:
 				case IFLA_STATS64:
 				case IFLA_STATS:
-				case IFLA_PROMISCUITY:
 				default:
 					break;
 			};
